terraform/terraform.tfvars

```diff
@@ -0,0 +1,3 @@
+env = "test"
+region = "us-east-1"
+api_spec_path = "../api_definition/coepi_api_0.3.0.yml"
```

Comment:
This needs to be updated with the latest spec version.

Reply:
I need terraform to create stacks for both v3 and v4. Is there a good way to re-use or inherit templates with different variables ?